generators/generator-botbuilder/generators/app/templates/echo/package.json.ts

```diff
@@ -27,6 +27,6 @@
         "@types/restify": "8.4.2",
         "nodemon": "^2.0.4",
         "tslint": "^6.1.2",
-        "typescript": "^3.9.2"
+        "typescript": "^4.3.2"
```

Comment:
Thanks for the PR @Sebastian-Rogers-BEIS, could we use 4.0.7 instead so other customers are not forced to update to 4.3.2?